UI/window-basic-settings.cpp

```diff
@@ -4970,6 +4973,11 @@ static void DisableIncompatibleCodecs(QComboBox *cbox, const QString &format,
 						 ? strEncLabel
 						 : obs_encoder_get_display_name(
 							   encoderId.c_str());
+
+		/* Something has gone horribly wrong and there's no encoder */
+		if (encoderId.empty())
```

Comment:
Is the first part of this comment accurate?
If we’re explicitly dealing with the situation because people regular-ish-ly run into it with reliable steps to reproduce, saying "something has gone horribly wrong" sounds confusing to me. (All this is assuming that this code is directly effected by problem)

Reply:
I'm just going to resolve this for now. We can sort out the details later if need be.